pages/[username]/comentarios/[page]/index.public.js

```diff
@@ -76,12 +88,20 @@ export const getStaticProps = getStaticPropsRevalidate(async (context) => {
 
   const secureContentListFound = authorization.filterOutput(userTryingToGet, 'read:content:list', contentListFound);
 
+  if (secureContentListFound.length === 0 && context.params.page !== 1) {
+    const lastValidPage = `/${secureUserFound.username}/comentarios``/${results.pagination.lastPage || 1}`;
```

Comment:
A sintaxe aqui está errada.

```suggestion
    const lastValidPage = `/${secureUserFound.username}/comentarios/${results.pagination.lastPage || 1}`;
```